plugins/module_utils/zoau_version_checker.py

```diff
@@ -96,16 +100,10 @@ def get_zoau_version_str():
     Returns:
         { [int, int, int] } -- ZOAU version found in format [#,#,#]. There is a
                                provision for a 4th level eg "v1.2.0.1".
+
     """
-    zoaversion_out = subprocess.run(
-        'zoaversion', shell=True, capture_output=True, check=False
-    )
     version_list = (
-        zoaversion_out
-        .stdout
-        .decode('UTF-8')
-        .strip()
-        .split(' ')[3][1:]
-        .split('.')
+        ZOAU_API_VERSION.split('.')
```

Comment:
@ketankelkar - can you comment on why you had the decode(utf-8) initially here that is being removed. I imagine the shell call to zoauversion is going to print out in the native locale which is going to be a EBCDIC code page, so wondering if there is any concern of this change. @richp405 feel free to also comment.

Reply:
Oh never mind, you are using the python api variable `ZOAU_API_VERSION`, sorry about that, i read to fast, it is nice subproccess is removed.